src/_sass/layout.scss

```diff
@@ -569,6 +569,34 @@ h3.relatedHeader {
   }
 }
 
+nav.sidebar h3 {
+  margin: 1rem 0 0.2rem 0;
+}
+
+.nav-accordion__panel[aria-hidden=true] {
+  display: none;
+}
+
+.nav-accordion__header[aria-expanded="true"]:before {
+  content: "- ";
+}
+
+.nav-accordion__header[aria-expanded="false"]:before {
+  content: "+ ";
```

Comment:
(optional nitpick) there's a little bit of layout shift that occurs with the character change from `+` to `-` and vice versa. i was able to avoid this by adding `font-family: monospace;` to both `:before` classes. removing the additional space after each character will maintain the same visual spacing you have now.